lib/bundler/cli.rb

```diff
@@ -807,5 +810,9 @@ def remembered_flag_deprecation(name)
         "do in future versions. Instead please use `bundle config #{name} " \
         "'#{value}'`, and stop using this flag"
     end
+
+    def metrics
```

Comment:
I think we want this method to be available on `Bundler` directly, like `Bundler.metrics`, so can you move this to `bundler.rb` and make it a class method?